app/javascript/dashboard/routes/dashboard/helpcenter/pages/portals/ListAllPortals.vue

```diff
@@ -69,6 +100,35 @@ export default {
     closeModal() {
       this.isAddModalOpen = false;
     },
+    onClickOpenDeleteModal(portal) {
+      this.selectedPortalForDelete = portal;
+      this.showDeleteConfirmationPopup = true;
+    },
+    async onClickDeletePortal() {
```

Comment:
Can we move this inside `portalItem`?

Reply:
Done https://github.com/chatwoot/chatwoot/pull/5305/commits/5e39942ddf476a040e09a4112da50a04f9dae94a